### PR TITLE
docs: Student report (computer)

### PR DESCRIPTION
**Pull request by TamkeenFatima**
# Checklist

- [ ] fixes #xxxx
- [x] Passes `Semantic Pull Request` and `TypoCheck`
- [ ] New docs are listed in `README.md`
- [ ] New docs are listed in `index.md`
- [ ] Updated table of content
- [ ] Updated nav links in mkdocs.yml

---

## PR description

Add Tamkeen's report
Update computer's `students.md`

### CODE REVIEW

File: docs/computer/student_Reports/Tamkeen_2019-20.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+### 1.5 Progress
+
+- As I am a complete beginner, I have started working on the prerequisites for the project. I learned C++ and am working on Java.
+- I plan to learn C# so that I can write my scripts for unity.  
+
+### 1.6 Future Works
+
+Incorporating real-time information about the vehicle such as battery level and the effects of pressure.  
+
+### 1.7 Refrences
```
Comment (pnijhara) [inline docs/computer/student_Reports/Tamkeen_2019-20.md]: typo

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
Could you please write stuff in 3rd person?

File: docs/computer/student_Reports/Tamkeen_2019-20.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- As I am a complete beginner, I have started working on the prerequisites for the project. I learned C++ and am working on Java.
+- I plan to learn C# so that I can write my scripts for unity.  
+
+### 1.6 Future Works
+
+Incorporating real-time information about the vehicle such as battery level and the effects of pressure.  
+
+### 1.7 References
+
+- Afzal, Afsoon & Katz, Deborah & Goues, Claire & Timperley, Christopher. (2020). A Study on the Challenges of Using Robotics Simulators for Testing.
```
Comment (syedjawadakhtar) [inline docs/computer/student_Reports/Tamkeen_2019-20.md]: A link to references would be better. Even if it is a research paper.

**Review by syedjawadakhtar — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
Are you on it @TamkeenFatima?

**Comment by pnijhara** [comment]
ping @TamkeenFatima !

**Review by pnijhara — APPROVED** [review]

**Pull request merged**